roles/katello_repositories/tasks/main.yml

```diff
@@ -9,3 +9,11 @@
 - name: enable codeready-builder for libdb_cxx used by qpid-cpp-server-linearstore
   command: subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms
   when: ansible_distribution_major_version == "8" and ansible_distribution == "RedHat"
+
+- name: 'Clean yum'
+  command: 'yum clean all'
+  args:
+    warn: false
+  tags:
+    - packages
+  when: ansible_os_family == 'RedHat'
```

Comment:
I think DNF does this correctly so should we limit this to EL7?

Reply:
We have the same structure in other modules.

Certainly can, but I guess that cleanup is needed in more places then :)